packages/schema/src/schema/geoTiffSource.json

```diff
@@ -22,7 +22,7 @@
         }
       },
       "minItems": 1,
-      "default": [],
+      "maxItems": 1,
```

Comment:
Doesn't that remove the need for a list?

Reply:
The GeoTiff sources option `sources` expects an array (https://openlayers.org/en/latest/apidoc/module-ol_source_GeoTIFF-GeoTIFFSource.html), but QGIS doesn't support multiple sources per layer the same way (as far as I know). So I felt a single item array was the best way to keep our schema in line with what OpenLayers wants while maintaining a QGIS-like experience, and provide some future-proofing if we want to support multiple sources per layer later on. 

(Also because everything using `urls` was written to use lists so it would mean a bunch of refactoring to change it)

Reply:
Ok sounds good 👍🏽 though how about removing the `maxItems: 1`? Otherwise we are not leaving the door open for: 
> and provide some future-proofing if we want to support multiple sources per layer later on.